services/user.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib.auth.hashers import make_password
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    user = User.objects.create_user(
+        username=username,
+        password=make_password(password),
+        email=email if email else "",
+        first_name=first_name if first_name else "",
+        last_name=last_name if last_name else ""
+    )
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> User | None:
+    try:
+        return User.objects.get(id=user_id)
+    except User.DoesNotExist:
+        print(f"User with user_id {user_id} does not exist")
+
+
+def update_user(
+        user_id: int,
+        username: str = None,
+        password: str = None,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> None:
+    try:
+        user = None
+        try:
+            user = User.objects.get(id=user_id)
+        except User.DoesNotExist:
+            print(f"User with user_id {user_id} does not exist")
```

Comment:
The try-except block for `User.DoesNotExist` is redundant here since the outer try-except block already handles this exception. Consider removing the inner try-except block and directly handling the exception in the outer block.